messenger.rst

```diff
@@ -1713,6 +1713,10 @@ The Beanstalkd transport DSN may looks like this:
 
 The transport has a number of options:
 
+``keepalive`` (default: none)
```

Comment:
The default keepalive interval is actually [5 seconds](https://github.com/symfony/symfony/blob/c0c0a8f625a8fdfee631efb34dc23500c961b3fb/src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php#L46). When the option is used without an explicit value, it activates the keepalive mechanism with that default value. You can optionally set a different interval by providing a value for the option.